docs/commands/add.md

```diff
@@ -5,40 +5,64 @@ title: modular add
 
 (i.e. `modular add my-app` would create a package in `packages/my-app`,
 `modular add @scoped/my-scoped-app` would create a package in
 `packages/my-scoped-app` and `modular add lib-a --path libs` would create a
 package in `libs/lib-a`)
 
-Packages can currently be one of the following types:
+The `modular add` command prompts the user to choose the Modular `type` of the
+package it's about to create. The next section briefly describes the various
+types that can be created by the `modular add` command. For an in-depth
+discussion of the available package types and their characteristics, please see
+[this page](../concepts/package-types.md).
 
-- A standalone `app`. This corresponds to a static Single Page Application (SPA)
-  project in a workspace. Inside this workspace, you can import packages from
-  other workspaces freely, and features like jsx and typechecking work out of
-  the box.
+### Standalone (bundled) package types
 
-- An `esm-view`, which is a package that typically exports a React component by
-  default. ESM Views are built as ES modules that can be `import`ed at runtime
-  by a host to implement a [micro frontend](../concepts/microfrontends.md)
-  architecture or started as a normal standalone application. See also
-  [the view building reference](../esm-views/index.md)
+These package types are built with [Webpack v5](https://webpack.js.org/) or, if
+specified in the [configuration](../configuration.md),
+[esbuild](https://esbuild.github.io/). Modules imported in the source these
+package types are bundled in the final result or, in case of `esm-view`s, they
+can be rewritten to an [external ESM CDN](../esm-views/esm-cdn.md).
 
-- A `view`, which is a `package` that exports a React component by default. Read
-  more about Views in [this explainer](../concepts/views.md).
+- `app`. This package type corresponds to a static Single Page Application (SPA)
+  project in a workspace. It's possible to specify a custom `index.html` file
+  and public asstets in the `public` directory.
 
-- A generic JavaScript `package`. You can use this to create a library with an
-  entry point that gets transpiled to Common JS and ES Module format when built.
-  Packages can be [built](../commands/build.md) but not
-  [start](../commands/start.md)ed by Modular.
+- `esm-view`. This package type is an app that gets built as an ES module that
+  can be imported at runtime. `esm-view`s are typically used to implement a
+  [micro-frontend](../concepts/microfrontends.md) architecture. `esm-views`,
+  when [built](./build.md) or [started](./start.md) will also generate a
+  `index.html` file that tries to load the ES Module and render its default
+  export as a React component onto the DOM (standalone mode). See also
+  [the esm-view reference](../esm-views/index.md) for an in-depth introduction.
+
+### Library package types
+
+These package types are either built with
+[Rollup.js](https://rollupjs.org/guide/en/) as CJS and ES Modules or, in case of
```

Comment:
Might be sensible to avoid an acronym for commonjs

```suggestion
[Rollup.js](https://rollupjs.org/guide/en/) as CommonJS and ES Modules or, in case of
```